tests/rptest/clients/kcl.py

```diff
@@ -247,25 +247,33 @@ def unfold(x):
         results = self._cmd(cmd, attempts=5).splitlines()[1:]
         results = [x for x in results if len(x) > 0]
 
-        def parse_line(line):
-            regex = re.compile(r"(\d+)\s+(.*)\s+(\d+)\s+(-?\d+)\s+(.*)")
-            error_regex = re.compile(r"(-?\d+)\s+(.*)")
-            matches = regex.match(line)
+        heading_regex = re.compile(r"BROKER.*")
+        result_regex = re.compile(r"(\d+)\s+(.*)\s+(\d+)\s+(-?\d+)\s+(.*)")
+        error_regex = re.compile(r"(-?\d+)\s+(.*)")
```

Comment:
nit: it'd help readability/maintainability to add a comment with some examples of the lines we're trying to match.

Reply:
This will be soon deprecated by the introducion of the `rpk trim` command